src/Config.php

```diff
@@ -25,4 +27,9 @@ public function getRules(): array
     {
         return array_merge($this->defaultRules, $this->extraRules);
     }
+
+    public function getCustomFixers(): array
+    {
+        return [new LineLengthLimit()];
```

Comment:
not really an issue but worth noting: rule has a default max length of 120, and won't auto-correct issues -- only flag them

Reply:
yeah, fixing should be possible but would be a hell of a lot of work to implement.

When i had a look at some of our services i think that 80 chars is too low, it would require us to make too many changes, 100 might be better but as the plugin does 120 by default i just left it at that thinking that any limit is better than nothing.